conf.py

```diff
@@ -71,7 +71,8 @@
     'sphinx.ext.todo',
     'sphinx.ext.mathjax',
     'opengraphext',
-    'sphinx_tabs.tabs']
+    'sphinx_tabs.tabs',
+    'sphinx_copybutton']
```

Comment:
how does it work? it adds copy button to all `code-block`s?

Reply:
yes, it's copy button in upper right corner